### PR TITLE
Support finding multiple elements

### PR DESCRIPTION
**Pull request by crabtw**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1245,6 +1251,14 @@ impl Client {
         self.by(search.into())
     }
 
+    /// Find elements on the page.
+    pub fn find_all(
+        &self,
+        search: Locator,
+    ) -> impl Future<Item = Vec<Element>, Error = error::CmdError> + 'static {
+        self.by_all(search.into())
```
Comment (jonhoo) [inline src/lib.rs]: Since nothing else needs `by_all`, I'd just place the contents of that method in here :)

**Comment by jonhoo** [comment]
Awesome! Could you please add a test or two for this feature?

**Comment by crabtw** [comment]
I have moved `by_all` into `find_all` and added one test.

File: src/lib.rs [hunk 2]
```diff
@@ -1981,6 +2015,22 @@ mod tests {
             })
     }
 
+    fn finds_all_inner<'a>(c: &'a Client) -> impl Future<Item = (), Error = error::CmdError> + 'a {
+        // go to the Wikipedia frontpage this time
+        c.goto("https://en.wikipedia.org/")
+            .and_then(move |_| c.find_all(Locator::Css("#p-interaction li")))
+            .and_then(move |es| {
+                assert_ne!(es.len(), 0);
```
Comment (jonhoo) [inline src/lib.rs]: I think I'd prefer a stronger assertion than this. If we want to future-proof the test for changes to wikipedia, maybe check that a few known items are in the list?

**Comment by crabtw** [comment]
It now checks the first 4 items.

**Comment by jonhoo** [comment]
Perfect, thank you!

**Pull request merged**